.travis.yml

```diff
@@ -7,8 +7,8 @@ env:
   matrix:
     - MOLECULE_DISTRO="centos:7"
     - MOLECULE_DISTRO="centos:8"
-    - MOLECULE_DISTRO="fedora:30"
```

Comment:
Is there some problem with fedora 30?

Reply:
I guess the idea is that it is unsupported, or soon will be. We are removing it form the test-harness CI as well.
For now lgtm, but in a longer term it would be preferable to use the template and do those changes centrally in the template.

Reply:
> Is there some problem with fedora 30?

There is... Somehow, python3 is used just for fedora 30??? (I thought it was because it was no longer supported... Maybe, my guess was wrong...)
https://travis-ci.org/github/linux-system-roles/logging/builds/689452565

Reply:
> > Is there some problem with fedora 30?
> 
> There is... Somehow, python3 is used just for fedora 30??? (I thought it was because it was no longer supported... Maybe, my guess was wrong...)
> https://travis-ci.org/github/linux-system-roles/logging/builds/689452565

It is not - https://travis-ci.org/github/linux-system-roles/logging/jobs/689452569 show that the same version of Python is used for fedora31. Why do you think the problem is in python3? 

> Hmmm, it might have been a temporary(?) failure. This run was successful... _puzzled_
> https://travis-ci.org/github/linux-system-roles/logging/builds/689530970

The actual error message is "ConnectionResetError: [Errno 104] Connection reset by peer" in ssl.py. This looks very much like a temporary network failure indeed, instead of being related to python3.

Could you please tell us why you need to do some change in the PR description instead of letting us guess?